python/lsst/ts/ess/common/sensor/boltek_efm100c.py

```diff
@@ -57,6 +58,15 @@ class Efm100cSensor(BaseSensor):
     both cases polarity will be +).
     """
 
+    def __init__(
+        self,
+        log: logging.Logger,
+    ) -> None:
+        super().__init__(log=log)
+
+        # Override default value.
+        self.charset = "UTF-8"
```

Comment:
This seems to be a class-wide value. If so, please consider using a class variable instead of an instance variable. In which case this `__init__` is not needed, and instead you just use:
```
class Foo(Parent):
   charset = "UTF-8"
```

Also...
* I gently suggest an enum, to avoid misspellings. Though the standard library doesn't seem to have one, which is a counter-argument.
* Why is this uppercase when python uses "utf-8" in library code? If it is necessary then a code comment explaining would be helpful.

Reply:
I promoted all instance variables to class variables where possible and got rid of all constructors in the sub-classes. And I reverted "UTF-8" to "ASCII".